src/app/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.html

```diff
@@ -20,12 +20,12 @@ <h5>{{ 'ubs-client-profile.personal' | translate }}</h5>
           <div class="form-group col-sm-5">
             <label for="recipientName">{{ 'ubs-client-profile.name' | translate }}</label>
             <input *ngIf="isEditing" type="text" class="form-control" id="recipientName" formControlName="recipientName" />
-            <p *ngIf="!isEditing">{{ userProfile?.recipientName }}</p>
+            <p *ngIf="!isEditing"></p>
```

Comment:
Why do we need this after deleting value?

Reply:
fixed